apitools/base/protorpclite/messages.py

```diff
@@ -1139,12 +1139,14 @@ def __setitem__(self, index, value):
 
     def append(self, value):
         """Validate item appending to list."""
-        self.__field.validate_element(value)
+        if getattr(self, '_FieldList__field', None):
+            self.__field.validate_element(value)
         return list.append(self, value)
 
     def extend(self, sequence):
         """Validate extension of list."""
-        self.__field.validate(sequence)
+        if getattr(self, '_FieldList__field', None):
```

Comment:
`hasattr` seems more appropriate? https://docs.python.org/3/library/functions.html#hasattr

Reply:
Yes, it would make sense but it would result in a divergent code path with the internal version of the library. Would it be fine to keep it in this current form?

Reply:
I see. I think it's fine.